### DISCUSSION TITLE
Publii is a great Static CMS preprocessor, but falls short on developer docs.

### DISCUSSION THREAD

**Discussion by nordseo** [GetPublii/Publii#1716] [Ideas]
There is not enough robust documentation in Publii docs. Can we start a collaborative repo for code examples of "How tos" ?

We have created some ourselves that have significantly improved Publii's functionality, such as breadcrumbs that _actually_ work and pass standards.

**Comment by candidexmedia** [comment]
Since the current docs don't allow for community contributions (might be too much of a hassle?), I think it's a good idea.

I've been maintaining my own personal db of useful forum answers, and have shared some in [my Gists](https://gist.github.com/candidexmedia).

Here are other sources I've collected so far for tips, tutorials, and third-party plugins:
- https://cv.domagic.site/
- https://p4p.pages.dev/
- https://fluegge.net/ohnein/tags/publii/
- https://plugliin.eu.org/tags/plugins/ 
- https://github.com/freaks-dev?tab=repositories
- https://github.com/bjazmoore?tab=repositories
- https://github.com/SimonPadbury/Publii-Bootstrap-Starter/tree/master/TUTORIALS

**Reply by treeandcoffee** [reply]
Great!!!
